ambari-server/src/main/java/org/apache/ambari/server/controller/internal/StackServiceResourceProvider.java

```diff
@@ -54,6 +54,9 @@
   protected static final String SERVICE_TYPE_PROPERTY_ID = PropertyHelper.getPropertyId(
 		  "StackServices", "service_type");
 
+  protected static final String SERVICE_CATEGORY_PROPERTY_ID = PropertyHelper.getPropertyId(
```

Comment:
Should also be added to `propertyIds`:

https://github.com/apache/ambari/blob/48d882d1b797cbb355a1cbc53389e2991b72ee04/ambari-server/src/main/java/org/apache/ambari/server/controller/internal/StackServiceResourceProvider.java#L117-L120